python/meep.i

```diff
@@ -1433,6 +1433,27 @@ void _get_gradient(PyObject *grad, PyObject *fields_a, PyObject *fields_f, PyObj
     $1 = NULL;
 }
 
+
+// typemaps for timing data
+
+%typemap(out) std::unordered_map<meep::time_sink, std::vector<double> > {
```

Comment:
Apparently using an `enum` as a hash key is a C++14 or later feature (https://stackoverflow.com/questions/18837857/cant-use-enum-class-as-unordered-map-key)?

Similar to #1687, you may need to pass `std::hash<int>` to `unordered_map` here to make sure this runs in Apple clang.  (Would be nice to have CI for this — if only to make sure things compile, even if we don't run the tests.)   See e.g. Argonne-National-Laboratory/neml#67 for a similar workaround in another project.

Reply:
This is not an enum class but a standard enum.

Reply:
Nevertheless, Apple clang is giving me an error:
```
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/c++/v1/type_traits:1848:38: error: implicit instantiation of undefined template 'std::__1::hash<meep::time_sink>'
    : public integral_constant<bool, __is_empty(_Tp)> {};
                                     ^
```

Reply:
That is really unexpected and annoying. Updated the code with `std::hash<int>`.